.github/workflows/test.yml

```diff
@@ -438,3 +470,69 @@ jobs:
   #         pixi-version: 0.1.0
   #     - run: exit 1
   #       if: success()
+
+  # test-frozen-and-locked:
+  #   runs-on: ${{ matrix.os }}
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       os: [windows-latest, ubuntu-latest, macos-latest]
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - name: Move pixi files
+  #       run: mv test/default/* .
+  #     - uses: ./
+  #       with:
+  #         pixi-version: v0.5.0
+  #         frozen: true
+  #         locked: true
+  #     - run: exit 1
+  #       if: success()
+
+  # TODO: this is an xfail right now. https://github.com/prefix-dev/pixi/issues/371
```

Comment:
Want to wait on this PR?

Reply:
Nah, I don't need to wait until your PR is merged. Since there is a fix now that will soon be merged, I can remove this comment.